client/Makefile.am

```diff
@@ -30,26 +27,22 @@ AM_CPPFLAGS =							\
 	$(SASL_CFLAGS)						\
 	$(POPT_CFLAGS)						\
 	$(WARN_CFLAGS)						\
-	$(INI_CFLAGS)						\
-	$(NULL)
+	$(INI_CFLAGS)
 
```

Comment:
The reason why is $(NULL) is at the end of list is simpler diff when adding new entries.

If there is not NULL at the end and you want to add new entry there then diff will contain 1 removed line + 2 added lines

If there *is* NULL at the end and  you want to add new entry there  then diff will contain just 1 added line.